### PR TITLE
[PCX-2161] New file header template

### PR DESCRIPTION
**Pull request by caiomello**
> Original PR: #133 

Added a new header template file that changes new file header comments to our custom format.

image

### CODE REVIEW

**Review by siebeprojects — APPROVED** [review]

File: PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+<key>FILEHEADER</key>
+<string> Copyright (c) ___YEAR___ Payoneer Germany GmbH
+// https://www.payoneer.com/
```
Comment (Sorix) [inline PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist:7]:
```suggestion
// https://www.payoneer.com
```

**Review by Sorix — CHANGES_REQUESTED** [review]

File: PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist (hunk 1 above)
Comment (caiomello) [reply PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist:7]: Can we _add_ the slash? It looks better... I'll happily do a replace to add it on every file.

Comment (Sorix) [reply PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist:7]: In my opinion, it doesn't look better. As well to do that, the header should be also replaced in other repositories using it. CC @siebeprojects

Comment (caiomello) [reply PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist:7]: Ah this is used in other projects? I didn't know. In that case, nevermind my suggestion.

Comment (Sorix) [reply PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist:7]: @caiomello, hmm, I still see the trailing slash, did you update it?

Comment (caiomello) [reply PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist:7]: Ah I rebased to remove a commit without the ticket so the change was lost (force push/rebase 🙄). I'll remove it again...

Comment (caiomello) [reply PayoneerCheckout.xcworkspace/xcshareddata/IDETemplateMacros.plist]: Done.

**Review by Sorix — APPROVED** [review]
✅, I will need to reapprove after you update it from the base branch. I think you could wait for PRs: #142, #140 to be merged.

**Comment by Sorix** [comment]
@caiomello, could you update the base branch? #142 will require QA so it will take time to complete it, so let's not rely on that PR.

**Review by siebeprojects — APPROVED** [review]

**Pull request merged**